.changeset/tidy-crabs-wash.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fuel-connectors/burner-wallet-connector": minor
+"@fuel-connectors/walletconnect-connector": minor
+"@fuel-connectors/evm-connector": minor
+"@fuel-connectors/fuel-wallet": minor
+---
+
+Converting address from bech32 to b256 on all connectors
```

Comment:
can we include a Breaking Change here? it can be smth like:

```
Breaking Change: now methods that return address, like "getAccount", will return a b256 address instead of a bech32.